cmd/apply/cmdapply.go

```diff
@@ -49,7 +49,7 @@ func NewCmdApply(f util.Factory, ioStreams genericclioptions.IOStreams) *cobra.C
 	cmdutil.AddValidateFlags(cmd)
 	cmd.Flags().BoolVar(&applier.ApplyOptions.ServerDryRun, "server-dry-run", applier.ApplyOptions.ServerDryRun,
 		"If true, request will be sent to server with dry-run flag, which means the modifications won't be persisted. This is an alpha feature and flag.")
-	cmd.Flags().Bool("dry-run", false,
+	cmd.Flags().BoolVar(&applier.ApplyOptions.DryRun, "dry-run", applier.ApplyOptions.DryRun,
```

Comment:
I think we should make dry-run an option that we pass as a parameter in the applier `Run` function. Would this be doable or will it require major refactoring?

Reply:
Let's stick with the way it's done in all the other CLI's--using a field in *Options. We can change it in the future.